overview.md

```diff
@@ -0,0 +1,110 @@
+# Ansible Distributed Continuous Integration Vendor Documentation
+
+## Overview
+
+Distributed Continuous Integration (*DCI*) testing is a system for vendors who maintain Ansible Network modules to host test equipment (switches, routers, etc.) for their modules in their own testing environments while providing an API and web interface for sharing the results of those tests with the Ansible maintainers. Network partners already have the necessary expertise and hardware/firmware matrix in house to be able to ensure compatibility with Ansible.This allows network vendors to ensure that their modules are correctly tested by giving them the ability to contribute comprehensive integration testing and test results.
+
+## Architecture
+
+DCI has two main components:
+
+1. A **control server** which provides a shared set of credentials, test results, and information about tests through a web application and REST API. This server is available as distributed-ci.io
+
+2. **DCI agent** which sits behind the vendor’s firewall and runs the integration tests.
+
+The control server is in charge of taking snapshots of the Ansible Git repo a number of times a day via the *feeder* process. It is configured to post these snapshots to *topics* which represent different releases (currently devel, and 2.3). The control server is managed by Ansible, and aside from getting access to the API, vendors don’t need to worry too much about this component.
+
+The DCI Agent, which triggers the tests locally. This can be done using cron jobs, or however the vendor sees fit. When this happens, the agent reaches out to the control server, authenticates, and pulls the latest snapshot for whichever topic (version) is being tested. The agent then uses Ansible playbooks to provision the correct test environment, runs the configured tests and then pushes the test results back to the control server. Once results are ready contributors from Ansible and the vendor can log in to the control server and see what happened.
```

Comment:
"The DCI agent is in charge of invoking integration tests. Since integration tests are just playbooks they can be run from the DCI agent periodically via cron jobs, manually or however the vendor sees fit."

Reword to fix weird sentence.